test/contrib/salesforce_test.py

```diff
@@ -88,7 +89,11 @@ def soql(self):
 
 
 class TestSalesforceQuery(unittest.TestCase):
-    @mock.patch('__builtin__.open', side_effect=mocked_open)
+    patch_name = '__builtin__.open'
+    if sys.version_info.major > 2:
+        patch_name = 'builtins.open'
+
+    @mock.patch(patch_name, side_effect=mocked_open)
```

Comment:
Well, tests passed. So if @Tarrasch approves, then i think this pr is good to go.